srsly/_json_api.py

```diff
@@ -39,6 +39,32 @@ def json_loads(data: Union[str, bytes]) -> JSONOutput:
     return ujson.loads(data)
 
 
+def json_loads_dict(data: Union[str, bytes]) -> Dict[str, Any]:
+    """Deserialize unicode or bytes to a Python dict.
+
+    data (str / bytes): The data to deserialize.
+    RAISES: ValueError if the loaded data is not a dict
+    RETURNS: The deserialized Python dict.
+    """
+    obj = json_loads(data)
+    if not isinstance(obj, dict):
+        raise ValueError("JSON data could not be parsed to a dict.")
+    return obj
+
+
+def json_loads_list(data: Union[str, bytes]) -> List[Dict[str, Any]]:
```

Comment:
This and `read_json_list` only checks `List` not `List[Dict[str, Any]]`? I wouldn't expect a function that's called `read_json_list` to have the additional dict behavior/type?

Reply:
Right this part is a little weird. Again, what I want 99% of the time when calling `read_json_list` is to have a list of dicts. But the naming does leave something to be desired. There is some validation in place for the `read_json_list` function, just not for the `json_loads_list`. I can copy the validation over there though.

Is `read_json_dicts` a better name here? similar to the `read_jsonl_dicts`.